Resources/doc/events.rst

```diff
@@ -0,0 +1,76 @@
+Registering Event Listeners and Subscribers
+===========================================
+
+Doctrine allows you to register listeners and subscribers that are notified
+when different events occur inside Doctrine's ODM. For more information,
+see Doctrine's `Event Documentation`_.
+
+.. note::
+
+    Each connection in Doctrine has its own event manager, which is shared with
+    document managers tied to that connection. Listeners and subscribers may be
+    registered with all event managers or just one (using the connection name).
+
+In Symfony, you can register a listener or subscriber by creating a service
+and then `tagging`_ it with a specific tag.
+
+*   **event listener**: Use the ``doctrine_mongodb.odm.event_listener`` tag to
```

Comment:
I'm not sure about using a bullet list in this case. Keep in mind that each bullet is super long and it even includes code samples ... so it's hard to see it as a part of a list. I'd suggest to reword this as sections or normal paragraphs.

Reply:
I'll change that to normal headings and text 👍